system variables: add docs for `signing_cert` and `signing_key` (#10549)
pingcap/docs: system-variables.md

</CustomContent>

> ### tidb_auth_signing_cert

Reviewer: not in v6.1

> ### tidb_auth_signing_cert

Reviewer: not for cloud

> ### tidb_auth_signing_cert <span class="version-mark">New in v6.2</span>

Reviewer: ```suggestion
### tidb_auth_signing_cert
```

- Scope: GLOBAL
- Persists to cluster: Yes
- Type: String
- Default value: ""
- This variable is associated with an unreleased feature. **DO NOT** set this variable.

> ### tidb_auth_signing_key

Reviewer: not for cloud

> - This variable is used in a feature this is not released yet. Please don't set this variable.

Reviewer: ```suggestion
- This variable is associated with an unreleased feature. Do not set this variable.
```

> ### tidb_auth_signing_key <span class="version-mark">New in v6.2</span>

Reviewer: ```suggestion
### tidb_auth_signing_key
```

- Scope: GLOBAL
- Persists to cluster: Yes
- Type: String
- Default value: ""

> - This variable is used in a feature this is not released yet. Please don't set this variable.

Reviewer: ```suggestion
- This variable is associated with an unreleased feature. **DO NOT** set this variable.
```
